defaults/main.yml

```diff
@@ -0,0 +1,44 @@
+---
+consul_version: 1.0.6
+
+consul_ip: "{{ ansible_default_ipv4.address }}"
```

Comment:
For some of our dedicated servers we may want to use the floating IP, at least for registering services, but we can use overrides when necessary.

Reply:
:+1: overriding them in the host variable files.